src/v2/Components/Auction/PostRegistrationModal.tsx

```diff
@@ -51,24 +49,26 @@ const RegistrationPendingUnverified: ModalContent = ({ onClick }) => {
         </a>{" "}
         or contact verification@artsy.net.
         <br />
-        <br />A link to complete identity verification has been sent to your
+        <br />
+        To complete your registration and start bidding, please click the{" "}
+        <strong>Verify identity</strong> button or follow the link sent to your
         email.
-      </Serif>
-      <ViewWorksButton onClick={onClick} />
+      </Text>
+      <OKButton onClick={onClick} />
```

Comment:
(nonblocking) Why wrap this in its own thing? The innerText including "OK" should be enough of a signal to future devs.

Reply:
Good point @damassi - i updated this in the pr where this commit comes from, will rebase here once that is merged.